src/applications/pensions/components/NoFormPage.jsx

```diff
@@ -157,7 +181,7 @@ export const NoFormPage = () => {
             visible
           >
             <h2 id="track-your-status-on-mobile" slot="headline">
-              You don’t have any saved online pension forms.
+              You don’t have any saved online pension form.
```

Comment:
Is this intentional? should "any" be changed to "an"

Reply:
intentional per approved design